101-cses-multi-role-rdp/azuredeploy.parameters.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "cloudServiceName": {
+      "value": "contosocses"
+    },
+    "location": {
+      "value": "centralus"
+    },
+    "deploymentLabel": {
+      "value": "ContosoCS_2021-01-21T005742"
+    },
+    "packageSasUri": {
+      "value": "https://gc18nov7stg.blob.core.windows.net/constocses/ContosoCS.cspkg?sp=r&st=2021-01-21T08:04:11Z&se=2021-01-21T16:04:11Z&spr=https&sv=2019-12-12&sr=b&sig=%2BCj%2B5hJrm9%2F4vKnOAobRsIlKG6gB%2FI4U9Y6OfLwhYfI%3D"
+    },
+    "configurationSasUri": {
+      "value": "https://gc18nov7stg.blob.core.windows.net/constocses/ServiceConfiguration.Cloud.cscfg?sp=r&st=2021-01-21T08:05:38Z&se=2022-01-01T16:05:38Z&spr=https&sv=2019-12-12&sr=b&sig=ADJT2Xuz8moVThcvMC4%2BQLJPAXcoam17%2FkgtLl8SeNk%3D"
+    },
+    "roles": {
+      "value": [
+        {
+          "name": "WebRole1",
+          "sku": {
+            "name": "Standard_D1_v2",
+            "capacity": "1"
+          }
+        },
+        {
+          "name": "WorkerRole1",
+          "sku": {
+            "name": "Standard_D1_v2",
+            "capacity": "1"
+          }
+        }
+      ]
+    },
+    "rdpPublicConfig": {
```

Comment:
for the config (and private config) - move the XML to a variable in the file and prompt for username and password as the param values.  Can also remove the \r\n

Asking the user to supply XML within JSON will be prone to failure.